pages/en/lb3/Managing-users.md

```diff
@@ -147,3 +147,18 @@ Since a model doesn't inherit ACLs from its base model, you must define ACLs for
 ### User realms
 
 See [Partitioning users with realms](Partitioning-users-with-realms.html).
+
+## Security considerations
+
+When user's account is compromised, for example when their password is leaked,
+or the attacker gains access to their email account, the user need means for
+recovering from the situation and preventing the attacker from continued use
+of the services under the attacked user's name.
```

Comment:
"When a user's account is compromised (for example their password is leaked or the attacker gains access to their email account), the app needs to be able to prevent continued use of the hijacked account."